internal/server/iigointernal/base_president.go

```diff
@@ -44,8 +51,9 @@ func (p *basePresident) EvaluateAllocationRequests(resourceRequest map[int]int,
 	return resourceAllocation, nil
 }
 
-// Chose a rule proposal from all the proposals
-// need to pass in since this is now functional for the sake of client side
+// pickRuleToVote choses a rule proposal from all the proposals made by the islands.
+// It returns an empty string if no rules were proposed, and the rule name (string)
+// of the selected rule otherwise.
 func (p *basePresident) PickRuleToVote(rulesProposals []string) (string, error) {
 	p.budget -= 10
```

Comment:
All the budget deductions in this PR for basespeaker, basepresident, and basejudge are placeholders as we haven't designed a proper costing system. Eventually we plan to have different 'service fees' for different actions. If it's alright with you, i'll add a comment but otherwise leave this as is because it will be overhauled soon anyhow,

Reply:
`10` seems to be hardcoded in many places, I think it would be good to replace this upfront, especially with #128 coming in which will standardise the type of `Resources`.